docs/ADLSGen2-HOWTO.md

```diff
@@ -30,3 +30,18 @@ Docs: [Create a storage account for Azure Data Lake Storage Gen2 | Microsoft Doc
 * [Blob service REST API - Azure Storage | Microsoft Docs](https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api)
 * [Azure Data Lake Storage Gen2 REST API reference - Azure Storage | Microsoft Docs](https://docs.microsoft.com/en-us/rest/api/storageservices/data-lake-storage-gen2)
 
+## Example
+
+When you want to connect to a deltatable stored in ADLS Gen2, you need to set two environment variables. 
+* AZURE_STORAGE_ACCOUNT_NAME which holds the name of the storage account 
+* AZURE_STORAGE_ACCOUNT_KEY which contains the root key for the storage account. 
+
+The url for the table should follow a specific format adls2://{accountname}/{filesystem}/{path to table}. 
+
+example:
+```python
+  from deltalake import DeltaTable
+
+  delta = DeltaTable("adls2://<accountname>/<filesystem>/<path to table>")
+  dataFrames = delta.to_pyarrow_table().to_pandas()
+```
```

Comment:
Thanks! Would you like to also add this to the [usage docs](https://delta-io.github.io/delta-rs/python/usage.html#loading-a-delta-table) as well?

It looks like I might have gotten the wrong prefix when I add this :facepalm::

https://github.com/delta-io/delta-rs/blame/main/python/docs/source/usage.rst#L59-L60

Reply:
Why use this non-standard uri schema for ADLS?
Should either use abfss if the underlying storage layer "knows" adls or use https with the appropriate blob storage endpoint that underlies every adls account.

Reply:
Actually, googling a bit, it seems like `abfss` is somewhat standard: https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction-abfs-uri
Thanks @zeevm 

@dgcaron Does that schema work for you?

Reply:
`abfs[s]` refers to the Hadoop File System driver (see the docs linked by @wjones127), which is not being used here.

Using `https` would be inconsistent with the fact that delta-rs accepts a URI with a cloud specific scheme ("s3://" and "gs://") for the Amazon and Google backends.

Reply:
> Why use this non-standard uri schema for ADLS? Should either use abfss if the underlying storage layer "knows" adls or use https with the appropriate blob storage endpoint that underlies every adls account.

delta-rs relies on the atomic rename functionality of ADLS Gen2 and cannot work with "plain" Blob Storage.